src/EdFi.AnalyticsMiddleTier.Tests/TestCases/StaffSectionDim/v_2/11324_XLTV31_should_have_StaffLastName.xml

```diff
@@ -4,11 +4,11 @@
   <ControlDataInsertion>
   </ControlDataInsertion>
   <Query>
-      SELECT FirstName
+      SELECT StaffLastName
```

Comment:
Looks like this was incorrect previously. Right? I mean, I changed from FirstName to StaffLastName.